processes/third_party_vuln_submit_form_hacker1.md

```diff
@@ -0,0 +1,59 @@
+> NOTE! Thanks for submitting a report! Please replace *all* the [square] sections below with the pertinent details. Remember, the more detail you provide, the easier it is for us to triage and respond quickly, so be sure to take your time filling out the report!
+
+I would like to report [VULNERABILITY] in [MODULE]
+It allows [DESCRIBE THE IMPACT OF THE VULNERABILITY - E.G READ ARBITRARY FILES, READ DATA FROM DATABASE ETC]
+
+# Module
+
+**module name:** [MODULE NAME]
+**version:** [MODULE VERSION]
+**npm page:** `https://www.npmjs.com/package/[MODULE NAME]`
+
+## Module Description
+
+> Copy description from npm page
+
+## Module Stats
+
+> Replace stats below with numbers from npm’s module page:
+
+Stats
+[1] downloads in the last day
+[10] downloads in the last week
+[100] downloads in the last month
+
+~[1200] estimated downloads per year [JUST FOR REFERENCE,  ~DOWNLOADS PER MONTH*12]
```

Comment:
I also suggest removing this line. I don't think it's useful or accurate, also anyone could calculate that manually if someone really needs it (I don't think so).

Calculating that number, copy-pasting it, and formatting that line takes extra steps while filling out this template, and that is already lengthy enough.

Reply:
I think it adds impact context but I can see how "more work" this can add for someone to add to the form when doing so manually.

I think the whole stats part can be automated once we hook up some API to H1 so this will be removed from the submitter responsibility entirely.